.gitignore

```diff
@@ -31,3 +31,5 @@ border.png
 profile.png
 lolsig.png
 /.vscode/launch.json
+sigma/plugins/nsfw/plugin.yml
```

Comment:
Why is this added to .gitignore?

Reply:
I only remember putting in config.yml so probably a mistake. Removed.